src/gui/CMakeLists.txt

```diff
@@ -103,6 +104,7 @@ transferlistmodel.cpp
 transferlistsortmodel.cpp
 transferlistwidget.cpp
 updownratiodialog.cpp
+uithememanager.cpp
```

Comment:
@jagannatharjun:  It is still in the wrong place.  These filenames should be listed in **alphabetical order**.  

In the English version of the Latin alphabet, `i` comes before `p`.  

```
updownratiodialog.cpp
uithememanager.cpp
```

…should be… 

```
uithememanager.cpp
updownratiodialog.cpp
```